src/App.js

```diff
@@ -1,23 +1,20 @@
-import logo from './logo.svg';
 import './App.css';
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import Header from './components/Header';
+import Books from './components/Books';
+import Categories from './components/Categories';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Router>
+        <Header />
+        <Switch>
+          <Route path="/" exact component={Books} />
+          <Route path="/categories" exact component={Categories} />
+        </Switch>
+      </Router>
```

Comment:
- [ ] The home page route should come after all routes, kindly start with route categories not home` `
![root](https://user-images.githubusercontent.com/55635977/131731797-cd4f6dcb-9570-43c5-9397-e0262023e3b8.png)

Reply:
Well solved